sw/device/lib/runtime/hart.c

```diff
@@ -11,7 +11,7 @@
 extern void wait_for_interrupt(void);
 
 void busy_sleep_micros(size_t microseconds) {
-  size_t cycles = kIbexClockFreqHz * microseconds / 10000000;
+  size_t cycles = kIbexClockFreqHz / 1000000 * microseconds;
   ibex_busy_loop(cycles);
```

Comment:
I think I'd make this int64_t arithmetic (and mark the literal `LL`) for overflow/underflow protection.